python_hb_report/utillib.py

```diff
@@ -0,0 +1,924 @@
+import os
+import datetime
+import sys
+import socket
+import re
+import envir
+import time
+import StringIO
+import subprocess
+import shutil
+import tempfile
+
+import xml.etree.ElementTree as ET
+from xml.dom import minidom
+
+from crmsh	import utils
+
+
+#set variables default
+def setvarsanddefaults():
+	'''
+	do some environment variable initial 
+	'''
+	now = datetime.datetime.now()
+	now_string = now.strftime("%Y-%m-%d %H:%M:%S.%f")
+	now_t = utils.parse_to_timestamp(now_string)
+	envir.UNIQUE_MSG="Mark:HB_REPORT:"+str(int(now_t))
+	NOW = now
+
+	envir.TO_TIME = utils.parse_to_timestamp(now_string)
+	date = datetime.datetime.date(now).strftime("%a-%d-%m-%Y")
+	envir.DEST = "hb_report-"+date
+	envir.SANITIZE.append("passw.*")
+	envir.SSH_OPTS = ['-o StrictHostKeyChecking=no','-o EscapeChar=none','-o ConnectTimeout=15']
+	envir.LOG_PATTERNS.append("CRIT:")
+	envir.LOG_PATTERNS.append("ERROR:")
+
+def debug(msg):
+	if (envir.VERBOSITY >0):
+		print >> sys.stderr,socket.gethostname(),"DEBUG:",msg
+		return 0
+
+def fatal(msg):
+	print >> sys.stderr,socket.gethostname(),"ERROR:",msg
```

Comment:
Not sure python3 support "print >> sys.stderr,socket.gethostname(),"ERROR:",msg" or not?